react-basic-practice/src/components/Button/index.tsx

```diff
@@ -28,7 +27,7 @@ const Button: React.FC<CustomButtonProps> = ({
     bg: "royalBlueLight",
     fontSize: "s",
     fontWeight: "medium",
-    borderRadius: borderRadius.xxs,
+    borderRadiuses: "xxs",
```

Comment:
Why use plural?

Reply:
Because this definition is correct for chakraUI's configuration, I checked this